source/Cosmos.Build.Builder/BuildTasks/CreateSetupTask.cs

```diff
@@ -41,7 +41,7 @@ protected override string GetExePath()
 
             if (!File.Exists(innoSetupCompilerPath))
             {
-                throw new InvalidOperationException("Inno Setup installation detected, but the compiler doesn't exist!");
+                throw new InvalidOperationException("An Inno Setup installation was detected, but the compiler doesn't exist!");
```

Comment:
Maybe add the path to the message, so that its easier to figure out the error when the message is shown.

Reply:
I don't know what the error output variable is for this specific area, do you know?

Reply:
> Maybe add the path to the message, so that its easier to figure out the error when the message is shown.

plus why would that part need a path output? it is just stating that inno compiler doesnt exist and needs to be installed

Reply:
I would add `innoSetupCompilerPath` to the message, since thats whats being checked for. IMO its equally likely that the compiler is not installed or the path is wrong, and if we can destinguish between the two that seems like a win.

Reply:
would this work?
![Screenshot 2021-07-16 150755](https://user-images.githubusercontent.com/76945439/126013030-693d016e-c8a0-40fa-95af-aed025455932.png)

Reply:
Yes thats what i had in mind. Although i think you are missing a $ before the string.

Reply:
> Yes thats what i had in mind. Although i think you are missing a $ before the string.

Alright, I've pushed it and it's doing appveyor thing